tsconfig.json

```diff
@@ -21,7 +21,10 @@
     ],
     "paths": {
       "@/*": ["./*"],
-      "@interfaces/*": ["./interfaces/*"]
+      "@interfaces/*": ["./interfaces/*"],
+      "@components/*": ["./app/components/*"],
+      "@hooks/*": ["./app/hooks/*"],
+      "@styles/*": ["./app/styles/*"],
```

Comment:
We use Tailwind so we don't have styles folder.

Reply:
resolved!